### PR TITLE
Search Endpoint added

### PR DESCRIPTION
**Pull request by zafertaskent**
/api/search/:text to search

### CODE REVIEW

File: api/controllers/DataDetailController.js [hunk 1]
```diff
@@ -7,13 +7,12 @@
...
 module.exports = {
 	findAll: function (req, res) {
-
 		DataDetail.find().populate('criteria').populate('indicator').populate('question').exec(function (err, datadetails) {
 			res.json(datadetails);
 		});
 	},
 	find: function (req, res) {
-		
+
```
Comment (adilatilgan) [inline api/controllers/DataDetailController.js]: Please clear extra space here

File: api/controllers/IndicatorController.js [hunk 2]
```diff
@@ -6,6 +6,15 @@
  */
 
 module.exports = {
-	
-};
 
+    search: function(req, res) {
+			
+        Indicator.find({
```
Comment (adilatilgan) [inline api/controllers/IndicatorController.js]: Why didn't you check  the text is null or invalid ?

Comment (zafertaskent) [inline api/controllers/IndicatorController.js]: This is done by sails default.

File: api/controllers/DataDetailController.js (hunk 1 above)
Comment (zafertaskent) [inline api/controllers/DataDetailController.js]: ok

**Comment by adilatilgan** [comment]
Ok LGTM

**Pull request merged**